### ISSUE TITLE
Check FederatedStore addGraph collision should not leak schema

### ISSUE THREAD

**Issue by GCHQDeveloper404** [gchq/Gaffer#1322]
Situation:

User1 adds private graph1
User2 adds graph1 (name conflict and error occurs)
Error message must not show the schema to users which do not have access to that graph.

**Comment by GCHQDeveloper404** [comment]
after #1365

**Comment by p013570** [comment]
Merged into develop.
